### PR TITLE
Count Key stack items inside Map items

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

File: src/neo-vm/ExecutionEngine.cs [hunk 1]
```diff
@@ -159,6 +159,7 @@ private static int GetItemCount(IEnumerable<StackItem> items)
                             if (counted.Any(p => ReferenceEquals(p, map)))
                                 continue;
                             counted.Add(map);
+                            count += map.Keys.Count; // Count key items
```
Comment (erikzhang) [inline src/neo-vm/ExecutionEngine.cs]:
```suggestion
                            count += map.Count; // Count key items
```

**Comment by erikzhang** [comment]
I think the following opcodes should be changed too: `SETITEM`, `APPEND`, and `REMOVE`.

**Comment by shargon** [comment]
What kind of change?

**Comment by erikzhang** [comment]
The invocations to `CheckStackSize()`. I'm not sure if they should be changed. Need to be checked.

**Comment by shargon** [comment]
Pending to find a best way for count items, but i will merge it because at the moment its fix a small bug

**Pull request merged**